lib/utils/getOverlayEntry.js

```diff
@@ -0,0 +1,57 @@
+const querystring = require('querystring');
+
+/**
+ * Creates an object that contains two entry arrays: the prependEntries and overlayEntries
+ * @param {import('../types').NormalizedPluginOptions} options Configuration options for this plugin.
+ * @param {import('webpack').Compiler.options.devServer | undefined} devServer The webpack devServer config
+ * @returns {import('./injectRefreshEntry').AdditionalEntries} An object that contains the Webpack entries for prepending and the overlay feature
+ */
+function getOverlayEntry(options, devServer) {
```

Comment:
This is a bit of a minor thing - but since we're generating `additionalEntries` here now, maybe it makes sense to rename this (and the file) to `getAdditionalEntries` or `getInjectedEntries`?